content/blog/persisting-kubernetes-workloads-with-amazon-efscsi-volumes-using-pulumi-sdks/index.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "Persisting Kubernetes workloads with Amazon EFS CSI volumes using Pulumi open source sdks"
```

Comment:
Nit: SDK is an acronym, so it should be capitalized as "SDKs".

Reply:
Done.